examples/time_frequency/plot_single_trial_spectra.py

```diff
@@ -35,48 +35,21 @@
 include = []
 raw.info['bads'] += ['MEG 2443']  # bads
 
-# picks MEG gradiometers
-picks = mne.pick_types(raw.info, meg='grad', eeg=False, eog=True,
-                       stim=False, include=include, exclude='bads')
-
-epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks, proj=True,
-                    baseline=(None, 0), reject=dict(grad=4000e-13, eog=150e-6))
-
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
+                    baseline=(None, 0),
+                    reject=dict(grad=4000e-13, eog=150e-6))
 
 n_fft = 256  # the FFT size. Ideally a power of 2
-psds, freqs = compute_epochs_psd(epochs, fmin=2, fmax=200, n_fft=n_fft,
-                                 n_jobs=2)
-
-# average psds and save psds from first trial separately
-average_psds = psds.mean(0)
-average_psds = 10 * np.log10(average_psds)  # transform into dB
-some_psds = 10 * np.log10(psds[12])
-
-
-fig, (ax1, ax2) = plt.subplots(1, 2, sharex=True, sharey=True, figsize=(10, 5))
 
-fig.suptitle('Single trial power', fontsize=12)
+# Let's first check out all channel types by averaging across epochs.
+epochs.plot_psds(plot_kind=1, fmin=2, fmax=200, n_fft=n_fft, n_jobs=2)
```

Comment:
@yousrabk the one would show an image, the other an averaged spectrogram plot, right? Not quite the same. 
Thinking about it maybe let's drop the image plot. Coverage is already good with plot_epochs_psds and plot_epochs_psds_topomap. Let's rathet tweak the topomap function to make it as nice as as possible.

Consider this example:

```Python
def plot_topomap_multi_cbar(data, pos, ax, title=None, unit=None,
                            vmin=None, vmax=None,
                            cmap='RdBu_r', colorbar=True):
    """Low level plot topography for a single measure

    Parameters
    ----------
    data : numpy.ndarray of float, shape (n_sensors,)
        The data show.
    pos : numpy.ndarray of float, shape (n_sensors, 2)
        The positions of the sensors.
    ax : instance of Axis
        The axes to plot on.
    title : str | None
        The axes title to show. Defaults to None (no title will be shown).
    
"""
    import matplotlib.pyplot as plt
    from mpl_toolkits.axes_grid1 import make_axes_locatable

    ax.set_yticks([])
    ax.set_xticks([])
    ax.set_frame_on(False)
    vmin = np.min(data) if vmin is None else vmin
    vmax = np.max(data) if vmax is None else vmax
    if title is not None:
        ax.set_title(title)
    im, _ = plot_topomap(data, pos, vmin=vmin,
                         vmax=vmax,
                         axis=ax,
                         cmap=cmap,
                         image_interp='bilinear',
                         contours=False)
    if colorbar is True:
        divider = make_axes_locatable(ax)
        cax = divider.append_axes("right", size="5%", pad=0.05)
        cbar = plt.colorbar(im, cax=cax, ticks=(vmin, vmax),
                            format='%3.3f')
        if unit is not None:
            cbar.set_label(unit)
        cbar.ax.tick_params(labelsize=8)

```